stats/rpc_constants.textpb

```diff
@@ -0,0 +1,620 @@
+# Constants for collecting specific RPC stats, including:
+#   - Measurement Descriptor, which describes a data point (measurement) type.
+#   - View Descriptor, which specifies an AggregationDescriptor and a set of tag keys.
+#     View Descriptor has one of the two kinds of Aggregation types associated with the View,
+#     Interval Aggregation Descriptor or Distribution Aggregation Descriptor.
+#     Aggregations are broken down by the unique set of matching tag values for each measurement.
+
+# Census defined RPC client MeasurementDescriptors.
```

Comment:
Should this say "gRPC defined" instead of "Census defined", since all of the names contain "grpc.io"?